src/examples/standard_tempo_cnn.cpp

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
Rename the file to `standard_tempocnn` so that it is easier to match with the name of the algorithm.

Reply:
it is done!